CMakeLists.txt

```diff
@@ -145,6 +145,7 @@ if (ENABLE_VECTOR_PREDICATE_SHORT_CIRCUIT)
   )
 endif()
 
+option(ENABLE_NETWORK_CLI "Allows use of the network cli" ON)
```

Comment:
The problem is, if we check in this PR, by default we will build the network-based client for the single-node version, no matter whether the machine has `grpc` installed or not. And typically, it will fail due to no `grpc` found.